linking/runtime-linking/link-function-at-runtime-on-windows.yml

```diff
@@ -6,8 +6,8 @@ rule:
       - moritz.raabe@mandiant.com
       - michael.hunhoff@mandiant.com
     scopes:
-      static: function
-      dynamic: unsupported  # requires characteristic features
+      static: instruction
+      dynamic: call
```

Comment:
with the optional part we're targeting multiple API calls which won't work this way

Reply:
ah, right...

I'm inclined to remove the `optional` and let this be used as a dependency more widely. Thoughts?

Reply:
or relax scope to `basic block`

Reply:
removed in 0c7d1bdfacac57cc6480b1e9f2e60019c1a130c7